### PR TITLE
Dev/mks 44 create end point for entity

### PR DESCRIPTION
**Pull request by FoxLviv**
create endPoint for entity

### CODE REVIEW

**Comment by Magfiozy** [comment]
Merge development\origin  into current branch before PR

File: Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
+            _userService = new UserService();
+        }
+
         // GET api/values
         [HttpGet]
-        public ActionResult<IEnumerable<string>> Get()
+        public IActionResult Get()
         {
-            return new string[] { "value1", "value2" };
+            //UserService userService = new UserService();
```
Comment (pytchak) [inline Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs]: Remove commented code

File: Andriy Burtso/TestAPI/TestAPI/Models/TestAPIContext.cs [hunk 2]
```diff
@@ -8,5 +8,6 @@ public class TestAPIContext: DbContext
         
         public DbSet<User> Users { get; set;  }
```
Comment (pytchak) [inline Andriy Burtso/TestAPI/TestAPI/Models/TestAPIContext.cs]: If you use ADO.NET remove Entity framework from solution

File: Andriy Burtso/TestAPI/TestAPI/Startup.cs [hunk 3]
```diff
@@ -22,10 +22,12 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
-            services.AddEntityFrameworkNpgsql().AddDbContext<TestAPIContext>(opt => opt.UseNpgsql(Configuration.GetConnectionString("TestAPIConnection")));
+           // services.AddEntityFrameworkNpgsql().AddDbContext<TestAPIContext>(opt => opt.UseNpgsql(Configuration.GetConnectionString("TestAPIConnection")));
             services.AddSwaggerGen(c =>
            {
                c.SwaggerDoc("v1", new Info { Title = "Core API", Description = "My Core API" });
+               //var XmlPath = System.AppDomain.CurrentDomain.BaseDirectory + @"TestAPI.xml";
```
Comment (pytchak) [inline Andriy Burtso/TestAPI/TestAPI/Startup.cs]: remove commented code

Comment (FoxLviv) [reply Andriy Burtso/TestAPI/TestAPI/Startup.cs]: done

File: Andriy Burtso/TestAPI/TestAPI/Models/TestAPIContext.cs (hunk 2 above)
Comment (FoxLviv) [reply Andriy Burtso/TestAPI/TestAPI/Models/TestAPIContext.cs]: done

**Review by pytchak — APPROVED** [review]

File: Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs [hunk 4]
```diff
@@ -1,30 +1,42 @@
...
+            _userService = new UserService();
+        }
+
         // GET api/values
         [HttpGet]
-        public ActionResult<IEnumerable<string>> Get()
+        public IActionResult Get()
         {
-            return new string[] { "value1", "value2" };
+            return Ok(_userService.GetList().ToList());
```
Comment (Magfiozy) [inline Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs]: Why u use ToList if your method already returns list?

File: Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs [hunk 5]
```diff
@@ -1,30 +1,42 @@
...
+
 
 namespace TestAPI.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class ValuesController : ControllerBase
     {
+        private readonly UserService _userService;
+        public ValuesController()
```
Comment (Magfiozy) [inline Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs]: It's not very important now but if you works with users please change naming.

**Review by TsisinskyjV — APPROVED** [review]

File: Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs (hunk 4 above)
Comment (FoxLviv) [reply Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs]: Done (I removed it)

File: Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs (hunk 5 above)
Comment (FoxLviv) [reply Andriy Burtso/TestAPI/TestAPI/Controllers/ValuesController.cs]: Renamed

**Review by Magfiozy — APPROVED** [review]

**Pull request merged**